tests/agent/python/flaskapp/smoke.py

```diff
@@ -0,0 +1,257 @@
+import time
+from tornado import ioloop, httpclient
+from elasticsearch import Elasticsearch
+from datetime import datetime, timedelta
+import subprocess
+import logging
+import json
+import copy
+import uuid
+
+p_uid = uuid.uuid4().hex
+
+num_reqs = 0    # use with care
```

Comment:
This comment somehow scares me, but I don't know why. Could you explain briefly why this should be used with care?

Reply:
right, ill explain better in the code 👍 
short story, because it is a global variable